pypyr-ops/Dockerfile

```diff
@@ -6,20 +6,38 @@
 # build me like this from repo root:
 # docker build -t pypyr/pypyr-ops -t pypyr/pypyr-ops:latest -t pypyr/pypyr-ops:1.0.0 -f pypyr-ops/Dockerfile .
 
-FROM pypyr/pypyr-go:pypyr1.1.1-go1.11.1
+FROM pypyr/pypyr-go:pypyr1.2.1-go1.11.2
 
-ARG terraformversion=0.11.8
+ARG terraformversion=0.11.10
 
-ARG awsversion=1.16.37
+ARG awsversion=1.16.48
 
-RUN  apt-get update && apt-get install -y apt-transport-https; \
-        wget https://releases.hashicorp.com/terraform/${terraformversion}/terraform_${terraformversion}_linux_amd64.zip; \
-         curl -s https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key add -; \
-         echo "deb http://apt.kubernetes.io/ kubernetes-xenial main" | tee -a /etc/apt/sources.list.d/kubernetes.list; \
+RUN  apt-get update && apt-get install -y apt-transport-https \
+        ca-certificates \
+        curl \
+        gnupg2 \
+        software-properties-common && \
+        wget https://releases.hashicorp.com/terraform/${terraformversion}/terraform_${terraformversion}_linux_amd64.zip && \
+         curl -s https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key add - && \
+         echo "deb http://apt.kubernetes.io/ kubernetes-xenial main" | tee -a /etc/apt/sources.list.d/kubernetes.list && \
+         curl -fsSL https://download.docker.com/linux/$(. /etc/os-release; echo "$ID")/gpg > /tmp/dkey && apt-key add /tmp/dkey && \
+         add-apt-repository \
+            "deb [arch=amd64] https://download.docker.com/linux/$(. /etc/os-release; echo "$ID") \
+            $(lsb_release -cs) \
+            stable" && \
          apt-get update && apt-get install -y unzip \
-         kubectl; \
-         unzip terraform_${terraformversion}_linux_amd64.zip; \
-         mv terraform /usr/local/bin; \
-         rm -f terraform_${terraformversion}_linux_amd64.zip; \
-         pip install awscli==${awsversion}; \
+         kubectl \
+         docker-ce && \
```

Comment:
simlarly, you can get the docker tar on a specific version from here: https://download.docker.com/linux/static/stable/x86_64/

the latest full url is https://download.docker.com/linux/static/stable/x86_64/docker-18.06.1-ce.tgz

i.e you can parameterize as `https://download.docker.com/linux/static/stable/x86_64/docker-{dockerversion}-ce.tgz`

then you'll want to untar something like `tar xzvf /path/to/<FILE>.tar.gz`
and then `mv` the `docker` file to bin. You're not interested in `dockerd`. 

clean up - `rm` the original tar and the extracted tar files you don't need

Reply:
Done